### ISSUE TITLE
Cannot create new roster (b24)

### ISSUE THREAD

**Issue by Millicant** [BattleScribe/iOS-Alphas#14]
I managed to create one roster in iOS. 

When I tried to create another, it fails. I hit the check mark to create the roster, the "creating roster" wheel shows up, but then it returns to the create roster page and no roster is created.

**Comment by Jonskichov** [comment]
I'm not able to reproduce this right now.

Was it related to the other startup/roster creation crashes that were happening before?

Is this still a problem?

**Comment by Millicant** [comment]
Yes this is still occurring. I'm testing with the Legiones Astartes catalogues from the Horus Heresy 2 repo.

**Comment by Jonskichov** [comment]
Can you give me specific steps to reproduce? Catalogues, entries selected etc etc
